windows/DateTimePickerWindows/DateTimePickerView.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "DateTimePickerView.g.h"
+#include "winrt/Microsoft.ReactNative.h"
+#include "NativeModules.h"
+
+namespace winrt::DateTimePicker::implementation {
+
+    class DateTimePickerView : public DateTimePickerViewT<DateTimePickerView> {
+    public:
+        DateTimePickerView(Microsoft::ReactNative::IReactContext const& reactContext);
+        void UpdateProperties(Microsoft::ReactNative::IJSValueReader const& reader);
+
+    private:
+        Microsoft::ReactNative::IReactContext m_reactContext{ nullptr };
+        bool m_updating{ false };
+        winrt::Windows::UI::Xaml::Controls::CalendarDatePicker::DateChanged_revoker m_dataPickerDateChangedRevoker{};
+
+        void RegisterEvents();
+        void OnDateChanged(winrt::Windows::Foundation::IInspectable const& sender, winrt::Windows::UI::Xaml::Controls::CalendarDatePickerDateChangedEventArgs const& args);
```

Comment:
alias it

Reply:
Should I create aliases for all namespaces I reference in this project?

Reply:
@rectified95 you should only need to create aliases for the top level namespaces that will change with WinUI3. See this file: https://github.com/microsoft/react-native-windows/pull/4766/files#diff-fed223fe01e306d5faa2fb1af7633252R98

namespace xaml = winrt::Microsoft::UI::Xaml;
namespace comp = winrt::Microsoft::UI::Composition;
namespace winrt {
namespace system = winrt::Microsoft::System;
using ColorHelper = winrt::Microsoft::UI::ColorHelper;
using Colors = winrt::Microsoft::UI::Colors;
} // namespace winrt

Reply:
Btw, I'm using winrt::**Windows**::UI::Xaml, not winrt::Microsoft::UI::Xaml - the WinUI3 PR changes the latter, so will this code be affected by it?

Reply:
right - you can just define your alias to point xaml = winrt::Windows::UI::Xaml and then use xaml. Then when we are ready to make this module WinUI3 compatible, we just need to change what the alias points to (instead of having to change every place where we use a Xaml type)